hazelcast/src/main/java/com/hazelcast/internal/diagnostics/HealthMonitor.java

```diff
@@ -180,9 +180,8 @@ private void logDiagnosticsHint() {
             // we only log the hint once
             performanceLogHint = false;
 
-            logger.info(String.format("The HealthMonitor has detected a high load on the system. For more detailed information,%s"
-                            + "enable the Diagnostics by adding the property -D%s=true",
-                    LINE_SEPARATOR, Diagnostics.ENABLED));
+            logger.info(format("The HealthMonitor has detected a high load on the system. For more detailed information,%n"
```

Comment:
`%n` uses the system line separator

Reply:
%n or \n ?

Reply:
I just did a small evaluation, and it seen %n workings like a line feed. I learned something new today.

Reply:
Exactly. When using `String.format()` IDEA should suggest `%n` when `\n` is used. In our case we provided the correct `LINE_SEPARATOR`, but `%n` is just much shorter.